fsevents.js

```diff
@@ -6,6 +6,13 @@
 /* jshint node:true */
 'use strict';
 
+if (process.platform !== 'darwin') {
+  var err = new Error('Cannot find module \'fsevents\'');
```

Comment:
I would reason the message itself is not being parsed by anyone, but I could see someone actually looking at `err.code` for `'MODULE_NOT_FOUND'`. This is why replicating with an actual `require` call was used.

As I said, I don't really care either way. I was just trying to reasonable mitigate potentially breaking existing dependents, which if occurred, they'd come looking at this repo for an explanation (and not me).

>I could imagine your latest suggestion still being confusing to people. They never tried to require fsevents-win32, and what happens when they go looking for it or attempting to install it?

So, someone on windows say, would directly try to use `fsevents`. So already they're quite confused before even writing the first line of code. They would then get the exception `fsevents-win32 not found`. They'd probably come to this repo, where either the README or some issue could clarify the situation.

However, all existing direct and indirect dependents currently out there would most likely keep working just fine, even those testing `err.code === 'MODULE_NOT_FOUND'`.

I would suggest using the code snippet and then seeing how the ecosystem responds. If there's a wall of confused people, at least nothing's breaking.

The objective of this PR is to remove the constant warning, without breaking existing dependents. But ultimately I will do whatever you require to accept a PR that stops the warning.

Can you stipulate your conditions for that?